templates/serve-stable-diffusion/tutotrials/walkthrough.md

```diff
@@ -0,0 +1,87 @@
+# Walkthrough of the template
+
+## Overview
+
+| Template Specification | Description |
+| ---------------------- | ----------- |
+| Summary | This app provides users a one click production option for serving a pre-trained Stable Diffusion model from HuggingFace.  It leverages [Ray Serve](https://docs.ray.io/en/latest/serve/index.html) to deploy locally and built in IDE integration on an Anyscale Workspace to iterate and add additional logic to the application. You can then use UI or a simple CLI to deploy to production with Anyscale Services. |
+| Time to Run | Around 2 minutes to setup the models and generate your first image(s). Less than 10 seconds for every subsequent round of image generation (depending on the image size). |
```

Comment:
is it intentional to advertise time in a separate line?

Reply:
I just copied this from the original template. We can remove it if we want..

Reply:
Removed this file completely. See the other comment.